src/esrs/event_bus/rabbit/config.rs

```diff
@@ -0,0 +1,49 @@
+use lapin::options::{BasicPublishOptions, ExchangeDeclareOptions};
+use lapin::types::FieldTable;
+use lapin::{BasicProperties, ConnectionProperties, ExchangeKind};
+use typed_builder::TypedBuilder;
+
+use crate::esrs::event_bus::rabbit::RabbitEventBus;
+use crate::event_bus::rabbit::error::RabbitEventBusError;
+use crate::Aggregate;
+
+#[derive(TypedBuilder)]
+pub struct RabbitEventBusConfig<'a> {
```

Comment:
Honestly not so sure about if this builder patter with `typed-builder` makes sense or is better to implement a manual builder..

Reply:
I think both are fine - I don't know what benefit manual would necessarily get us here, except maybe some build time benefit? (I don't know if the build time grows with options with this library)

Reply:
Since this builder has 9000 fields I don't see why not use `typed-builder`?